src/IECoreDelight/Renderer.cpp

```diff
@@ -1505,6 +1505,24 @@ class DelightRenderer final : public IECoreScenePreview::Renderer
 			const V2i &resolution = camera->getResolution();
 			screeenParameters.add( { "resolution", resolution.getValue(), NSITypeInteger, 2, 1, NSIParamIsArray } );
 
+			const int overscanLeft = static_cast<int>(camera->getOverscanLeft() * resolution.x);
+			const int overscanTop = static_cast<int>(camera->getOverscanTop() * resolution.y);
+			const int overscanRight = static_cast<int>(camera->getOverscanRight() * resolution.x);
+			const int overscanBottom = static_cast<int>(camera->getOverscanBottom() * resolution.y);
+
+			const Box2i overscan(
+				V2i(
+					overscanLeft,
+					overscanTop
+				),
+				V2i(
+					overscanRight,
+					overscanBottom
+				)
+			);
+
+			screeenParameters.add( { "overscan", overscan.min.getValue(), NSITypeInteger, 2, 2, NSIParamIsArray } );
+
 			Box2i renderRegion = camera->renderRegion();
 
 			// I can't find any support in 3delight for overscan - and if crop goes outside 0 - 1,
```

Comment:
This old comment seems to be inaccurate now. It also seems that the old handling of crop windows isn't consistent with the new overscan code (crop is intended to be measured with respect to the non-overscanned image), which you can see if you compare to an Arnold render. I think fixing that could be a bit involved so let's not tackle it in this PR - what you've done already represents good progress.

Could you just update the old comment to say this?

```
// If crop goes outside 0 - 1, 3Delight ignores it, so we clamp.
/// \todo 3Delight interprets crop as 0-1 across the overscanned data window, but
/// Gaffer defines it as 0-1 across the original (non-overscanned) display window.
/// Adjust for that. This will only work nicely with the CropWindowTool if we also
/// update the display driver to output an accurate display window for overscanned renders.
```

Reply:
Absolutely no problem. Comment updated.